nativeio_async_wrapper.js

```diff
@@ -180,6 +192,45 @@ mergeInto(LibraryManager.library, {
     })
   },
 
+  NativeIO_SetLength__deps: ['$NativeIOWrapper','$NativeIOUtils', '$Asyncify'],
+  NativeIO_SetLength: function(fileDescriptor, {{{ defineI64Param('length') }}}) {
+    return Asyncify.handleSleep(function(wakeUp) {
+      {{{ receiveI64ParamAsI32s('length') }}}
+      try {
+        var length = NativeIOUtils.combineI64Params(length_low, length_high);
+      } catch (error) {
+        console.log(
+          'NativeIO error receiving length to truncate file with file descriptor number',
+           fileDescriptor + ':', error);
+        wakeUp(-{{{cDefine('EINVAL')}}});
+      }
+
+      NativeIOWrapper.setLength(fileDescriptor, length).then(
+        () => {wakeUp(0)},
+        (error) => {
+          console.log(
+            'NativeIO error while setting length of file with file descriptor number',
+            fileDescriptor, 'to', offset + ':', error);
```

Comment:
Nit: you could use ` for string formatting here as well

Reply:
Done, as well as doing it for all lines were params were used within an error message.